### ISSUE TITLE
Server returned no JSON data when trying to retrieve building footprints

### ISSUE THREAD

**Issue by nebnebnebneb** [gboeing/osmnx#158]
First of all great repository! I hope to harness your package in my master's thesis but I am fairly new to python so I might have missed something obvious. 

I was trying to follow your example (no. 10) and met with the error when trying to retrieve building footprints from Piedmont. 

I ran the code on jupyter, with python 2 and OSMnx version 0.7.2. The packages in my environment are as attached: [packages_b.txt](https://github.com/gboeing/osmnx/files/1963157/packages_b.txt)

```python
ExceptionTraceback (most recent call last)
<ipython-input-4-40571a0b7fcd> in <module>()
----> 1 gdf = ox.buildings_from_place(place='Piedmont, California, USA')
      2 gdf_proj = ox.project_gdf(gdf)
      3 fig, ax = ox.plot_buildings(gdf_proj, bgcolor='#333333', color='w', save=True, show=False, close=True, filename='piedmont_bldgs', dpi=40)
      4 Image('{}/{}.{}'.format(img_folder, 'piedmont_bldgs', extension), height=size, width=size)

C:\Users\B\Anaconda3\lib\site-packages\osmnx\buildings.pyc in buildings_from_place(place, retain_invalid)
    283     city = gdf_from_place(place)
    284     polygon = city['geometry'].iloc[0]
--> 285     return create_buildings_gdf(polygon, retain_invalid=retain_invalid)
    286 
    287 

C:\Users\B\Anaconda3\lib\site-packages\osmnx\buildings.pyc in create_buildings_gdf(polygon, north, south, east, west, retain_invalid)
    159     """
    160 
--> 161     responses = osm_bldg_download(polygon, north, south, east, west)
    162 
    163     vertices = {}

C:\Users\B\Anaconda3\lib\site-packages\osmnx\buildings.pyc in osm_bldg_download(polygon, north, south, east, west, timeout, memory, max_query_area_size)
    125                               '(poly:"{polygon}")["building"];(._;>;));out;')
    126             query_str = query_template.format(polygon=polygon_coord_str, timeout=timeout, maxsize=maxsize)
--> 127             response_json = overpass_request(data={'data':query_str}, timeout=timeout)
    128             response_jsons.append(response_json)
    129         msg = ('Got all building footprints data within polygon from API in '

C:\Users\B\Anaconda3\lib\site-packages\osmnx\core.pyc in overpass_request(data, pause_duration, timeout, error_pause_duration)
    358             else:
    359                 log('Server at {} returned status code {} and no JSON data'.format(domain, response.status_code), level=lg.ERROR)
--> 360                 raise Exception('Server returned no JSON data.\n{} {}\n{}'.format(response, response.reason, response.text))
    361 
    362         return response_json

Exception: Server returned no JSON data.
<Response [400]> Bad Request
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
    "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
<head>
  <meta http-equiv="content-type" content="text/html; charset=utf-8" lang="en"/>
  <title>OSM3S Response</title>
</head>
<body>

<p>The data included in this document is from www.openstreetmap.org. The data is made available under ODbL.</p>
<p><strong style="color:#FF0000">Error</strong>: line 1: parse error: ';' expected - ')' found. </p>
<p><strong style="color:#FF0000">Error</strong>: line 1: parse error: Unexpected end of input. </p>
<p><strong style="color:#FF0000">Error</strong>: line 1: static error: Element "print" cannot be subelement of element "union". </p>

</body>
</html>
```

Thank you so much!

**Comment by temetski** [comment]
I'm getting the same error here. But on a script that I have gotten to work one week ago. I did not change anything in my script, downloading graphs works fine. It's downloading buildings that triggers this error for me. osmnx 0.7.4 here

**Comment by gboeing** [comment]
@nebnebnebneb @temetski looks like something changed with the API. Just updated the query string in https://github.com/gboeing/osmnx/commit/094d63c77aae016073350d081dd660c252397b62 and pushed to master. That should fix it. Please test and let me know.

**Comment by nebnebnebneb** [comment]
@gboeing It works now, thank you so much!

**Comment by gboeing** [comment]
See also https://github.com/drolbr/Overpass-API/issues/480

**Comment by gboeing** [comment]
Fix released in v 0.8.
